### PR TITLE
Load AWS E2E creds from secret manager

### PR DESCRIPTION
**Pull request by Elad-Laufer**

### CODE REVIEW

File: build/action.yml [hunk 1]
```diff
@@ -34,6 +37,13 @@ runs:
         secrets: |-
           GITHUB_ACTIONS_SSH_READ_KEY:${{ inputs.gcp-project-id }}/${{ inputs.ssh-key-secret-name }}
 
+    - id: e2e-secrets
+      uses: google-github-actions/get-secretmanager-secrets@v0
+      with:
+        secrets: |-
+          GITHUB_ACTIONS_E2E_AWS_CRED:${{ inputs.gcp-project-id }}/${{ inputs.e2e-aws-cred }}
```
Comment (golanpa) [inline build/action.yml:44]: where GITHUB_ACTIONS_E2E_AWS_CRED is used?

File: build-and-push/action.yml [hunk 2]
```diff
@@ -47,6 +50,13 @@ runs:
           GITHUB_ACTIONS_AWS_ACCESS_KEY_ID:${{ inputs.gcp-project-id }}/${{ inputs.aws-key-id-secret-name }}
           GITHUB_ACTIONS_AWS_SECRET_ACCESS_KEY:${{ inputs.gcp-project-id }}/${{ inputs.aws-key-secret-secret-name }}
 
+    - id: e2e-secrets
+      uses: google-github-actions/get-secretmanager-secrets@v0
+      with:
+        secrets: |-
+          GITHUB_ACTIONS_E2E_AWS_CRED:${{ inputs.gcp-project-id }}/${{ inputs.e2e-aws-cred }}
```
Comment (golanpa) [inline build-and-push/action.yml:57]: where GITHUB_ACTIONS_E2E_AWS_CRED is used? 
I also see that you are using:

- name: Configure AWS credentials
      uses: aws-actions/configure-aws-credentials@v1
      with:
        aws-access-key-id: ${{ steps.secrets.outputs.GITHUB_ACTIONS_AWS_ACCESS_KEY_ID }}
        aws-secret-access-key: ${{ steps.secrets.outputs.GITHUB_ACTIONS_AWS_SECRET_ACCESS_KEY }}
        aws-region: us-east-1

but GITHUB_ACTIONS_AWS_ACCESS_KEY_ID and GITHUB_ACTIONS_AWS_SECRET_ACCESS_KEY are not declared in the secrets step

**Review by golanpa — CHANGES_REQUESTED** [review]

File: build/action.yml (hunk 1 above)
Comment (Elad-Laufer) [reply build/action.yml:44]: In builds that have e2e tests such as:

https://github.com/wix-private/mcloud-go-framework
https://github.com/wix-private/media-platform-goppelganger
https://github.com/wix-private/media-platform-upload-server

File: build-and-push/action.yml (hunk 2 above)
Comment (Elad-Laufer) [reply build-and-push/action.yml:57]: Because they are used to push the image to ECR and not to run tests.

**Review by golanpa — APPROVED** [review]

**Pull request merged**